.renovaterc.json

```diff
@@ -17,6 +17,6 @@
     }
   ],
   "lockFileMaintenance": {
-    "enabled": true
```

Comment:
this is a huge pain and not worth it

Reply:
I might agree but can you elaborate? This would have caught us just later, right?

Reply:
I'm just not a fan of having my work day hijacked by 🤖 s. I don't like renovate because it makes updating dependencies seem "easier" than it really should be.

Reply:
This is the second week in a row that I had to manually take over the renovate yarn.lock PR...

Reply:
> I'm just not a fan of having my work day hijacked by 🤖 s.

I see but it hijacks you work day once a week.

> This is the second week in a row that I had to manually take over the renovate yarn.lock PR...

True that sounds no fun. Still if you loose a yarn.lock after not updating it for 5 months sounds like less fun even.

Reply:
I have no strong opinion one way or another, so I guess let's try turning it off and see how it goes.